packages/spectral/src/testing.ts

```diff
@@ -399,7 +399,7 @@ const createConfigVars = <TConfigVarValues extends TestConfigVarValues>(
         ...result,
         [key]: {
           ...value,
-          key,
+          key: value.key,
```

Comment:
Wouldn't we accomplish the same mission by deleting this line since we're spreading `value`?